docs/tutorials/gymnasium_basics/environment_creation.py

```diff
@@ -276,7 +228,7 @@ def reset(self, seed=None, options=None):
 # environment after applying that action and returns the 5-tuple
 # ``(observation, reward, terminated, truncated, info)``. See
 # :meth:`gymnasium.Env.step`. Once the new state of the environment has
-# been computed, we can check whether it is a terminal state and we set
+# been computed, we can check whether it is a terminal state, and we set
```

Comment:
Remove the comma, change to "and then set ~~``done``~~ ``terminated``(?) accordingly.